plugin.yaml

```diff
@@ -1,11 +1,11 @@
 name: "scan2html"
-repository: github.com/afdesk/scan2html
-version: "0.0.1"
+repository: github.com/malikovmm/scan2html
```

Comment:
we don't have to rename the repository.
i understand that you did it for tests. but, please, clean up the PR